src/uws_client.c

```diff
@@ -568,6 +568,7 @@ void uws_client_destroy(UWS_CLIENT_HANDLE uws_client)
             for (i = 0; i < uws_client->protocol_count; i++)
             {
                 free(uws_client->protocols[i].protocol);
+                uws_client->protocols[i].protocol = NULL;
             }
 
             free(uws_client->protocols);
```

Comment:
does this need a null assignment as well? Seams like they are crying about it in the other places.

Reply:
It only seems to be complaining about the ones that are inside of arrays, but I could add it here too to be safe!